src/devices/immax.ts

```diff
@@ -227,6 +227,16 @@ const definitions: Definition[] = [
             ],
         },
     },
+    {
+        fingerprint: [{modelID: 'TS004F', manufacturerName: '_TZ3000_krwtzhfd'}],
+        model: 'TS004F-Immax',
```

Comment:
```suggestion
        model: '07767L,
```

Took this from their site